### DISCUSSION TITLE
after `trigger('phone')` then error wont clear automatic after typing

### DISCUSSION THREAD

**Discussion by amantiwari1** [react-hook-form/react-hook-form#6036] [Q&A]
![image](https://user-images.githubusercontent.com/53963065/126549957-4ba83104-3a60-4808-8877-bfcedfece186.png)

step 1 - I tapped `verify` button for `trigger('phone')` and got error below input
step 2 - type valid number
step 3 - still error and it is clear valid still 

any solution?

**Comment by bluebill1049** [comment]
provide a simple codesandbox.

**Reply by amantiwari1** [reply]
Sorry!, here codesanbox - https://codesandbox.io/s/errormessage-name-type-forked-nenu7?file=/src/index.tsx

Step 1 -  Tap Trigger Button
Step 2 - type something in the input

mine expected to clear error automatic after type at least 1 or more letter

result

not clear error automatic after typing

**Reply by bluebill1049** [reply]
change mode to 'onChange'

**Reply by amantiwari1** [reply]
can you example?

**Reply by bluebill1049** [reply]
useForm mode: onChange
